en/sql/tuning.rst

```diff
@@ -4189,3 +4191,166 @@ The user can check the query to be cached or not by putting the session command
     }
 
 The cached query is shown as **query_string** in the middle of the result screen. Each of the **n_entries** and **n_pages** represents the number of cached queries and the number of pages in the cached results. The **n_entries** is limited to the value of configuration parameter **max_query_cache_entries** and the **n_pages** is limited to the value of **query_cache_size_in_pages**. If the **n_entries** is overflown or the **n_pages** is overflown, some victims among the cache entries are selected and they are uncached. The number of victims is about 20% of **max_query_cache_entries** value and of the **query_cache_size_in_pages** value.
+
+.. _correlated-subquery-cache:
+
+SUBQUERY CACHE (correlated)
+------------------------------------
+
+Subquery cache optimization can enhance the performance of queries including correlated subqueries by caching each subquery's results in separate spaces. 
+This optimization is enabled by default. 
+To disable it during query execution, use the NO_SUBQUERY_CACHE hint on the target subquery.
+
+If the correlated subquery is in the SELECT clause, subquery cache is utilized. 
+The operational procedure involves using cached results when the values of columns referenced in a rerun correlated subquery match those in the main query. 
```

Comment:
```suggestion
If the column values ​​of the search conditions of the re-executed correlated subquery are the same, the cached results are used instead of executing the subquery.
```